### ISSUE TITLE
Can't submit to App Store with 1.10.0

### ISSUE THREAD

**Issue by rzulkoski** [uias/Tabman#331]
### New Issue Checklist

- [x] Updated Tabman to the latest version
- [x] I took a look at the [Troubleshooting Guide](https://github.com/uias/Tabman/blob/master/Docs/TROUBLESHOOTING.md)
- [x] I searched for [existing GitHub issues](https://github.com/uias/Tabman/issues)

### Issue Description

When attempting to upload to App Store Connect I get the following errors:

```
ERROR ITMS-90685: "CFBundleIdentifier Collision. There is more than one bundle with the CFBundleIdentifier value 'com.uias.AutoInsetter' under the iOS application 'Storefront.app'."

ERROR ITMS-90685: "CFBundleIdentifier Collision. There is more than one bundle with the CFBundleIdentifier value 'com.uias.Pageboy' under the iOS application 'Storefront.app'."

ERROR ITMS-90205: "Invalid Bundle. The bundle at 'Storefront.app/Frameworks/Tabman.framework' contains disallowed nested bundles."

ERROR ITMS-90206: "Invalid Bundle. The bundle at 'Storefront.app/Frameworks/Tabman.framework' contains disallowed file 'Frameworks'."
```

I have determined that the cause of the errors is the new `Copy Carthage Frameworks` shell script build phase in the Tabman project.

I can confirm that I am able to submit a build successfully using 1.9.2.

### Other useful things

I am using Xcode 10, Swift 4.2, macOS 10.14 Mojave

A different framework that had a similar problem:
https://github.com/ably/ably-ios/issues/698

The pull request that solved their problem:
https://github.com/ably/ably-ios/pull/700/files

**Comment by rzulkoski** [comment]
Here is the Tabman commit that appears to have introduced the problem:
https://github.com/uias/Tabman/commit/9ca2ed5824d6b962a1237595ad11dc2c5859263e#diff-2a68191693b6e1d2a38aafd7ac0bcca9

**Comment by msaps** [comment]
@rzulkoski thanks once again for all the help! Should be fixed as of [1.10.1](https://github.com/uias/Tabman/releases/tag/1.10.1)
